app/models/transaction.rb

```diff
@@ -1,4 +1,6 @@
 class Transaction < ApplicationRecord
+  MIN_DATE = Date.today - 1095
```

Comment:
Remove.

Reply:
Please note that `MIN_DATE` here is calculated when the application is booted up and is execute only once.  As a constant, it will not change until the app is restarted.